docs/source/usage/firstwrite.rst

```diff
@@ -239,46 +239,51 @@ Python
 Units
 -----
 
-Ouch, our measured magnetic field data is in `Gauss <https://en.wikipedia.org/wiki/Gauss_(unit)>`_!
-Quick, let's store the conversion factor to SI (`Tesla <https://en.wikipedia.org/wiki/Tesla_(unit)>`_).
+Let's decribe this magnetic field :math:`\vec B` in more detail.
+Independent of the absolute unit system, a magnetic field has the `physical dimension <https://en.wikipedia.org/wiki/Dimensional_analysis>`_ of [mass (M)\ :sup:`1`, electric current (I)\ :sup:`-1`, time (T)\ :sup:`-2`].
```

Comment:
Thank you for this PR. This is really good as is. (The only thing I would wonder, but really not important, would be to use dots instead of commas...)

Reply:
offline discussion: so instead of a vector of base quantities `[M^1, I^-1, T^-2]` a "product" of the kind `[M^1 * I^-1 & T^-2]`.

Reply:
This seems perfect to me! Thanks